src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -90,8 +90,8 @@ module sonic-feature{
                 leaf check_up_status {
                     description "This configuration controls the system ready tool to check
                         the app ready/up status";
-                    type boolean;
-                    default false;
+                    type stypes:boolean_type;
+                    default "false";
```

Comment:
Add unit test?

Reply:
@qiluo-msft Without this fix sonic-utilities submodule update is blocked. There were no UT when this field was added. I suggest to take this fix to unblock submodule update and the owner of new feature adds UT.

Reply:
@wen587 for awareness of blocking issue of sonic-utilities submodule update